lib/qbwc/request/inventoryassemblyproducts.rb

```diff
@@ -0,0 +1,85 @@
+module QBWC
+  module Request
+    class Inventoryassemblyproducts
```

Comment:
This needs to be `InventoryAssemblyProducts` unless there's a very good reason to make me squint like that

Reply:
No - all Request classes are based on payload types. The payload is `inventoryassemblyproduct` so when we capitalize, we get the class Name